src/openapi.rs

```diff
@@ -0,0 +1,44 @@
+use utoipa::OpenApi;
+use utoipa_axum::router::OpenApiRouter;
+
+#[derive(OpenApi)]
+#[openapi(
+    info(
+        title = "crates.io",
+        description = "API documentation for the [crates.io](https://crates.io/) package registry",
+        terms_of_service = "https://crates.io/policies",
+        contact(name = "the crates.io team", email = "help@crates.io"),
+        license(),
+        version = "0.0.0",
```

Comment:
the OpenAPI spec does not specify the format of the `version` field other than that it's a "string". we could potentially use the git SHA1 or commit timestamp as the `version`, if that seems useful 🤔

Reply:
I thinking a combination of both would probably be most informative: something like `2024-12-11-21-45-00-4b33fdda250e7a8ebb4c40eff4d6205ce09c7e2a`.